_config.yml

```diff
@@ -4,11 +4,11 @@ title                    : "Earth Lab Resources"
 title_separator          : "-"
 name                     : "Earth Analytics"
 description              : "Code, tutorials, and tools for modern Earth analytics"
-url                      : "https://earthlab.github.io"
+url                      : #"https://earthlab.github.io"
```

Comment:
please note - i'm testing removing this BECAUSE i have a feeling that travis is trying to building using the external link - IE our live site rather than a local link whereever it builds. i'm not sure how to force this in the build.